kadai-4/src/pages/dashboard.vue

```diff
@@ -77,10 +94,16 @@
     },
     data() {
       return {
-        userList: [],
         showContent: false,
         displayUserName:'',
-        displayWalletData:''
+        displayWalletData:'',
```

Comment:
細かいですが、`:`の後ろはスペースを空けるのがjsの記述作法となっております。
```suggestion
        displayWalletData: '',
```
他の箇所も全て同様です。

エディタの自動整形の機能を使用すると、このようなフォーマットのミスが無くせますので、取り入れてみると良いと思います。
https://qiita.com/mitashun/items/e2f118a9ca7b96b97840